torch/fx/experimental/symbolic_shapes.py

```diff
@@ -1824,6 +1824,29 @@ def print_results(grouped, indent, result_fn):
 TLS = threading.local()
 
 
+@dataclass(frozen=True)
+class CreateSymbolicSizesStridesStorageOffsetInput:
```

Comment:
would it make sense to name this something more generic if we anticipate it encompassing things like Sparse tensors in the future ? (i dont know, asking)